### ISSUE TITLE
[Webform] Recaptcha causes Siteimprove Errors (HTML is used to format content, Text area has no description, iFrame is missing a description)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#883]
On our webforms, the Recaptcha causes the following Siteimprove errors:
- HTML is used to format content
- Text area has no description
- iFrame is missing a description
